.github/workflows/weekly_dependency_test.yml

```diff
@@ -7,7 +7,7 @@ name: Weekly Dependency Test
 on:
   workflow_dispatch:
   schedule:
-    - cron: '0 22 * * mon' # each Monday night, at 22:00 UTC
+    - cron: '0 22 * * THU' # each Thursday night, at 22:00 UTC
```

Comment:
Is there a reason you changed this to all caps?

Reply:
Caps seems to be the default and VSCode marks lowercase as wrong.